libs/ui/src/lib/benchmark/benchmark.component.html

```diff
@@ -6,6 +6,31 @@
     </td>
   </ng-container>
 
+  <ng-container matColumnDef="date">
+    <th
+      *matHeaderCellDef
+      class="d-none d-lg-table-cell text-right"
+      mat-header-cell
+    >
+      <span class="d-none d-sm-block text-nowrap" i18n>Last All Time High</span>
+    </th>
+    <td
+      *matCellDef="let element"
+      class="d-none d-lg-table-cell text-right"
+      mat-cell
+    >
+      <gf-value
+        class="d-inline-block justify-content-end text-primary"
+        [isDate]="true"
+        [locale]="locale"
+        [value]="
+          element?.performances?.allTimeHigh?.date
+            | date: 'dd.MM.yyyy' ?? undefined
```

Comment:
I think the date pipe is not needed at all because of `[isDate]="true"`

Reply:
without the date pipe, date is getting displayed in mm/dd/yyyy format.

Reply:
yes

Reply:
The date is formatted in the [value component](https://github.com/ghostfolio/ghostfolio/blob/main/libs/ui/src/lib/value/value.component.ts#L106) to your local date format.